om4labs/om4common.py

```diff
@@ -462,10 +479,26 @@ def read_topography(dictArgs):
         ds = cat["topog"].to_dask()
 
     if "deptho" in list(ds.variables):
-        depth = ds.deptho.to_masked_array()
+        depth = ds.deptho
     elif "depth" in list(ds.variables):
-        depth = ds.depth.to_masked_array()
+        depth = ds.depth
+
+    if outputgrid == "symetric":
+        """ extend array, assuming periodic/tripolar grid"""
+        warnings.warn("assuming the grid is periodic and tripolar")
+        tmp = depth.values
+        inner = xr.DataArray(tmp, dims=("y", "x"))
+
+        # southern boundary
+        south = xr.DataArray(tmp[0, :], dims=("x"))
```

Comment:
you can also use this routine: https://github.com/raphaeldussin/static_downsampler/blob/3f7c0354e907cb420ff67bc97c6f4c053b3b0715/static_downsampler/static.py#L185